docs/data-collection/tear-down.md

```diff
@@ -78,7 +78,7 @@
 - [ ] **Retrieve ET recordings** (from {{ secrets.hosts.psychopy | default("███") }}):
     - [ ] Insert a USB key into *{{ secrets.hosts.psychopy | default("███") }}* and save the experiment from AcqKnowledge.
     - [ ] Upload to a pre-designated drop-box (e.g., using Dropbox)
-
+- [ ] Press ‘CTRL+ALT+Q’ on the ET PC Tower to exit the EyeLink 1000 Plus Host PC application and click on the Shutdown button from the File Manager tool bar.
```

Comment:
Have you tested this on our setup? From what I remember, when I exited the Eyelink application, I was left with a black screen, so "click on the Shutdown button from the File Manager toolbar" would not have been possible.